src/app/components/listing/confirm/confirm-developer.jsx

```diff
@@ -351,15 +359,17 @@ function ChplConfirmDeveloper(props) {
                       </MenuItem>
                     ))}
                   </ChplTextField>
+                  <br />
                   { !isShowingComparison
                     && (
-                      <CardActions>
-                        <Button
-                          onClick={() => setIsShowingComparison(true)}
-                        >
-                          Compare with uploaded information
-                        </Button>
-                      </CardActions>
+                    <Button
+                      variant="contained"
+                      color="secondary"
+                      onClick={() => setIsShowingComparison(true)}
+                    >
+                      Compare with uploaded information
+                      <CompareArrowsIcon className={classes.iconSpacing} />
```

Comment:
It occurs to me that we might not need the `classes.iconSpacing` thing, if the only thing we're trying to do is put a space between the text and the icon. Can you see if this construct works, and if so, maybe we can stop using this class?

```suggestion
                      Compare with uploaded information
                      {' '}
                      <CompareArrowsIcon />
```

Reply:
The code you provide didnt create enough spacing for me.